bot.py

```diff
@@ -362,8 +362,15 @@ def recipient_color(self):
             cmd = message.content[len(prefix):].strip()
             if cmd in self.snippets:
                 message.content = f'{prefix}reply {self.snippets[cmd]}'
+    
+        ctx = await self.get_context(message)
+        if ctx.command:
+            return await self.invoke(ctx)
+
+        thread = await self.threads.find(channel=ctx.channel)
 
-        await self.process_commands(message)
+        if thread is not None:
+            await self.modmail_api.append_log(message, type='internal')
     
```

Comment:
Even though there isn't a command being invoked here, the message should still propagate through discord. Or else it might mask issues in the future.

```suggestion
await self.process_commands(message)
```

Reply:
I mean add it at the end of `on_message`, that "suggested change" section might be a bit misleading.

Reply:
But we are not logging command messages